apps/remix-ide/src/app/components/plugin-manager-settings.js

```diff
@@ -2,8 +2,8 @@ const yo = require('yo-yo')
 const csjs = require('csjs-inject')
```

Comment:
If this file is no longer in use, please delete file.

Reply:
Removed as its not being used.